src/sentry/plugins/bases/notify.py

```diff
@@ -90,6 +108,11 @@ def get_sendable_users(self, project):
         return member_set
 
     def should_notify(self, group, event):
+        # If digests are enabled for this project, we always want to add the
```

Comment:
Muted groups are also filtered out when the digest is constructed, so this allows the notification to still be sent if the group is unmuted before the digest is sent. I don't have a strong opinion on this either way (and you could make a pretty reasonable argument to the contrary) but this approach seemed a little more flexible.